python_bindings/test/apps/local_laplacian_generator.py

```diff
@@ -0,0 +1,224 @@
+"""
+Fast image interpolation using a pyramid.
```

Comment:
This comment looks like it's from the interpolate app

Reply:
oopsie, fixing